src/layouts/MyLayout.vue

```diff
@@ -4,110 +4,116 @@
+      v-model="rightDrawerOpen"
+      side="right"
       bordered
       content-class="bg-grey-2"
     >
       <q-list>
         <q-item-label header>
-          Essential Links
+          [USERNAME]
         </q-item-label>
         <q-item
           clickable
           tag="a"
-          target="_blank"
-          href="http://v1.quasar-framework.org"
+          :to="{ name: 'home' }"
         >
           <q-item-section avatar>
-            <q-icon name="school" />
+            <q-icon name="home" />
           </q-item-section>
           <q-item-section>
-            <q-item-label>Docs</q-item-label>
+            <q-item-label>Home</q-item-label>
             <q-item-label caption>
-              v1.quasar-framework.org
+              Main Page
             </q-item-label>
           </q-item-section>
         </q-item>
         <q-item
           clickable
           tag="a"
-          target="_blank"
-          href="https://github.com/quasarframework/"
+          :to="{ name: 'user', params: { userId: 456, slug: 'john-cena' }}"
         >
           <q-item-section avatar>
-            <q-icon name="code" />
+            <q-icon name="person" />
           </q-item-section>
           <q-item-section>
-            <q-item-label>Github</q-item-label>
+            <q-item-label>My Page</q-item-label>
             <q-item-label caption>
-              github.com/quasarframework
+              User Profile
             </q-item-label>
           </q-item-section>
         </q-item>
         <q-item
           clickable
           tag="a"
-          target="_blank"
-          href="http://chat.quasar-framework.org"
+          :to="{ name: 'new' }"
         >
           <q-item-section avatar>
-            <q-icon name="chat" />
+            <q-icon name="cloud_upload" />
           </q-item-section>
           <q-item-section>
-            <q-item-label>Discord Chat Channel</q-item-label>
+            <q-item-label>Upload</q-item-label>
             <q-item-label caption>
-              chat.quasar-framework.org
+              New Meme
             </q-item-label>
           </q-item-section>
         </q-item>
         <q-item
           clickable
           tag="a"
-          target="_blank"
-          href="https://forum.quasar-framework.org"
+          :to="{ name: 'settings' }"
         >
           <q-item-section avatar>
-            <q-icon name="record_voice_over" />
+            <q-icon name="settings" />
           </q-item-section>
           <q-item-section>
-            <q-item-label>Forum</q-item-label>
+            <q-item-label>Settings</q-item-label>
             <q-item-label caption>
-              forum.quasar-framework.org
+              Profile Settings
             </q-item-label>
           </q-item-section>
         </q-item>
         <q-item
           clickable
           tag="a"
-          target="_blank"
-          href="https://twitter.com/quasarframework"
+          :to="{ name: 'home' }"
```

Comment:
instead of linking `:to=...` someplace, this button should trigger a function. You can use `@click="logout'` for example, and add `logout` as a new function under `methods` in the `<script>` area. Stub it out with an `alert()` to show it's getting called.